cli-main.js

```diff
@@ -174,7 +179,21 @@ if (options.nodeVersion) {
 }
 
 (async () => {
-	if (options.stdin) {
+	if (options.printConfig) {
+		if (input.length > 0) {
+			console.error('The `print-config` option must be used with exactly one file name');
```

Comment:
```suggestion
			console.error('The `--print-config` flag must be used with exactly one filename');
```

Reply:
This message is confusing as it makes it sound like the user passed multiple filenames to `--print-config`.

Reply:
That’s exactly the mistake this is intended to catch: `xo --print-config x.js y.js`. (This is parsed as `{flags: {printConfig: 'x.js'}, input: ['y.js']}`, but a user who writes that probably doesn’t know that.)